docs/docs/img/UML_new.svg

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?> 
```

Comment:
Couple of things wrong here, let's discuss it on teams on wednesday/friday.

Reply:
Renewed by Julia